opensquirrel/ir.py

```diff
@@ -13,6 +13,15 @@
 from opensquirrel.common import ATOL, are_matrices_equivalent_up_to_global_phase, normalize_angle
 
 
+_REPR_DECIMALS = 5
+
+
+def _repr_round(
+        value: float | Axis | NDArray[np.complex64],
+        decimals: int = _REPR_DECIMALS) -> float | Axis | NDArray[np.complex64]:
+    return np.round(value, decimals)
+
+
```

Comment:
I leave this up to you, but I would move this code to utils/math.py, and I would add a few basic tests for this function: round a float, an axis, and an NDArray, and pass no decimals, 0, 1, and -1.

Reply:
If you check the `np.round` docs, this is what they say for a negative decimal input:

> If decimals is negative, it specifies the number of positions to the left of the decimal point.